projects/demo/project.json

```diff
@@ -8,6 +8,8 @@
         "build": {
             "executor": "@angular-devkit/build-angular:browser",
             "options": {
+                "baseHref": "/",
```

Comment:
The most base case:

* For local development
* For Firebase live preview channels
* For e2e
* For other hostings (not gh-pages)